frontend/src/ui/player/Paella.tsx

```diff
@@ -275,9 +273,9 @@ const PAELLA_CONFIG = {
     },
 };
 
-const tracksToPaellaSources = (tracks: VideoTrack[], isLive: boolean): Stream["sources"] => {
-    const trackToSource = (t: VideoTrack): Source => {
-        const [w, h] = t.resolution;
+const tracksToPaellaSources = (tracks: Track[], isLive: boolean): Stream["sources"] => {
+    const trackToSource = (t: VideoTrack | Track): Source => {
```

Comment:
```suggestion
    const trackToSource = (t: Track): Source => {
```

`Track` is already an audio or video track, right? Like `VideoTrack` is just a subset of `Track`, right?